app/workers/downstream_live_worker.rb

```diff
@@ -81,4 +85,14 @@ def enqueue_dependencies
   def notify_airbrake(error, parameters)
     Airbrake.notify(error, parameters: parameters)
   end
+
+  def send_ga_event
+    uri = URI.parse('http://www.google-analytics.com/collect')
+    params = { 'v': '1', 'tid': 'UA-26179049-1', 'cid': '660ad712-9753-4cb9-97a7-c9e9f13c318e', 't': 'event', 'cd90': edition.first_published_at }
+    Net::HTTP.post_form(uri, params)
+  end
+
+  def first_time_being_published?
+    %w(major minor).include? edition.update_type
```

Comment:
`published_or_unpublished` was always returning an edition because the point where the GA event had to be sent was after an edition had been published. 

I'm using this to check if there's already a published edition present: 

`edition.document.editions.where(state: 'substituted').count == 0`